### DISCUSSION TITLE
[Feature] shift-select multiple photos

### DISCUSSION THREAD

**Discussion by martibc16** [immich-app/immich#4545] [Feature Request]
### The feature

In any file explorer the user has the option to select a range of items by clicking shift+first_photo and shift+last_photo. This feature would be very useful in immich web because then we could select multiple photos from multiple days without having to select each day.
Simply shift select the first photo and the last one and every photo in between will be selected.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
You can already do this from the web

**Comment by jrasm91** [comment]
Duplicate of #3550
